payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AfterpayClearpayHeaderElement.kt

```diff
@@ -21,16 +21,32 @@ data class AfterpayClearpayHeaderElement(
 
     val infoUrl = url.format(Locale.current.region.lowercase())
 
-    fun getLabel(resources: Resources) =
-        resources.getString(
-            R.string.afterpay_clearpay_message,
-            CurrencyFormatter.format(
-                amount.value / 4,
-                amount.currencyCode
+    fun getLabel(resources: Resources): String {
+        val numInstallments = when (amount.currencyCode.lowercase()) {
+            "eur" -> 3
```

Comment:
Do we have enums for currency?

Reply:
No, currently we don't.  There is a java.util.Currency class that we could use to get strongly typed currencies.  Right now I can see it being good to strongly type the currency in the Amount.kt, CurrencyFormatter.  Not sure there is a lot of value in adding it for the older classes.

Given the size of this type of change I would recommend pulling it out as a tech debt item.